apps/ui/src/components/MessageErrorFetchPower.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+defineProps<{
+  type: 'proposition' | 'voting';
+}>();
+
+defineEmits<{
+  (e: 'fetch');
+}>();
+</script>
+
+<template>
+  <div class="flex flex-col gap-3 items-start" v-bind="$attrs">
+    <UiAlert type="error">
+      There was an error fetching your {{ type }} power.
```

Comment:
Maybe we need to display similar error on proposals page as well
![image](https://github.com/user-attachments/assets/c141c9f0-286c-4968-a7c9-9ef99982cba2)

But maybe in different PR

Reply:
Ah i mean for voting validation, we check it in proposals page as well